.travis.yml

```diff
@@ -0,0 +1,15 @@
+---
+sudo: false
+language: ruby
+cache: bundler
+
+rvm:
+  - 2.5.1
```

Comment:
FYI InSpec standard is 2.3.latest, 2.4.latest, 2.5.latest

Reply:
For reference here's what we use in our Azure resource pack: https://github.com/inspec/inspec-azure/blob/master/.travis.yml#L6-L9